### DISCUSSION TITLE
Vue için justified gallery çözümü

### DISCUSSION THREAD

**Discussion by gorkie** [ademilter/homepage#40] [Q&A] [answered]
Merhaba,

Vue içerisinde kullanacağım yatay rowlardan oluşan ve justified şekilde hizalanmış bir galeri grid yapısı oluşturmak istiyorum ancak vue ile uyumlu sorun çıkarmayan bir plugin bulamadım. Aşağıdaki örnek linkte olacak şekilde bir yapıyı nasıl hazırlayabilirim? Yada vue component içerisinde kullanabileceğim benzer bir yapı var mıdır? 

Örnek yapı:
https://depositphotos.com/search/pizza.html

Bu konuda bana yardımcı olabilirseniz çok sevinirim, teşekkürler.

Görkem.

**Comment by ademilter** [comment]
Merhaba görkem,
Örnek verdiğin sayfadaki yapmak istediğin şeyin ekran görüntüsünü atabilir misin. Sayfada bir yatay kategori alanı var bir de içerik alanındaki grid. Hangisini kasettin tam olarak

**Comment by gorkie** [comment]
Merhaba,

İçerik alanından bahsediyorum. Aşağıya ekran görüntüsünü olarak ekledim. 

<img width="1592" alt="screen" src="https://user-images.githubusercontent.com/16632488/48678900-bd67fd80-eb9a-11e8-96c8-7c7f08b949a8.png">

**Comment by ademilter** [comment, marked as answer]
Bununla alakalı okuduğum en iyi kaynak şu yazı
https://link.medium.com/fAC6ACclXR

İlk aklıma gelen bu olduğu için yazıyorum. Eklentiye denk gelirsem tekrar yazarım

**Comment by gorkie** [comment]
Çok teşekkürler, bende hemen inceleyeceğim.
